renderc/io.opentelemetry.render/src/io/opentelemetry/render/generator/WriterGenerator.xtend

```diff
@@ -252,14 +252,13 @@ class WriterGenerator {
     '''
     #pragma once
 
-    #include <generated/«app.pkg.name»/«app.name»/encoder.h>
+    #include "encoder.h"
+    #include "parsed_message.h"
+    #include "wire_message.h"
 
     #include <channel/ibuffered_writer.h>
     #include <platform/types.h>
 
-    #include <generated/«app.jb_h»>
-    #include <generated/«app.jsrv_h»>
```

Comment:
After this change, does the code reference jb_h, jsrv_h, descriptor_h? If not maybe we should remove as to not confuse a future reader

Reply:
Yes, marked for removal.